### ISSUE TITLE
Add an option to stop receiving stream for node maintenance

### ISSUE THREAD

**Issue by FranckUltima** [livepeer/go-livepeer#2976]
Problem

Currently, when performing maintenance on a node, all streams to that node must be stopped. This can be disruptive to users who are currently watching streams.

Solution

Add an option to the Livepeer client to stop receiving streams for node maintenance.

Implementation

The option could be implemented as a flag in the Livepeer client. When the flag is set, the client would stop receiving new streams. Existing streams would continue to play until they are finished.

Benefits

This feature would improve the quality of service for Livepeer users by reduce the disruption to users caused by node maintenance.

**Comment by Tibia2000** [comment]
Just set price per pixel to maximum and you wont receive new streams while retaining current ones

**Comment by FranckUltima** [comment]
True, but in the future with the multiplication of broadcasters and different prices, I find that an option in the CLI would be more attractive.
